src/TagsInput/components/DeleteButton.js

```diff
@@ -4,25 +4,24 @@ import * as React from 'react';
 
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 import { Touchable } from '../../Touchable';
-import type { StylePropType } from '../../PlatformStyleSheet/StyleTypes';
 
 import { Icon } from '../../Icon';
 
 type Props = {|
   +onPress: () => void,
-  +style: StylePropType,
+  +opacity: number,
```

Comment:
I would still leave the style as prop, it will be more extensible in the future.

Reply:
I am still not convinced by that. It is not consistent with the rest of the codebase. If I want to reuse that component, I should be able to. I don't see why we should prevent re-use.